### ISSUE TITLE
Backend Server start failed

### ISSUE THREAD

**Issue by fredo340** [derekhe/msfs2020-map-enhancement#608]
Hi, I'm having the following issues. I have disable my antivirus, reinstall, followed the FAQ. but still the same.
![ERROR](https://github.com/derekhe/msfs2020-map-enhancement/assets/167651924/26d17ba0-c6af-47d4-b456-c8e69a04569f)
![Command Prompt](https://github.com/derekhe/msfs2020-map-enhancement/assets/167651924/d6e0b60a-9e43-4f44-9c70-e7fb2d3ffd8a)
please help...

**Comment by derekhe** [comment]
Please try the solution there: https://stackoverflow.com/questions/10461257/an-attempt-was-made-to-access-a-socket-in-a-way-forbidden-by-its-access-permissi

**Comment by fredo340** [comment]
here is what I found, not sure that this is what is creating the problem,

![Port Excluded](https://github.com/derekhe/msfs2020-map-enhancement/assets/167651924/5acb7421-f810-487c-86dc-b64e8b6f7e86)
port 443 is excluded on my PC

![Try to Delete Exclusion](https://github.com/derekhe/msfs2020-map-enhancement/assets/167651924/99c8038e-3bca-42b2-8ee6-5a5f01a085b4)
But when trying to delete the exclusion, I'm being denied.

could that be the problem?

Thanks

**Comment by fredo340** [comment]
still can't get it to work

**Comment by fredo340** [comment]
please help, I've install 7.5.16 but still same

**Comment by skulhd** [comment]
good day getting the same error on version .15 and .16

**Comment by ozcan181** [comment]
> here is what I found, not sure that this is what is creating the problem,
> 
> ![Port Excluded](https://private-user-images.githubusercontent.com/167651924/324351913-5acb7421-f810-487c-86dc-b64e8b6f7e86.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxNzIxMjYsIm5iZiI6MTcxNDE3MTgyNiwicGF0aCI6Ii8xNjc2NTE5MjQvMzI0MzUxOTEzLTVhY2I3NDIxLWY4MTAtNDg3Yy04NmRjLWI2NGU4YjZmN2U4Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNlQyMjUwMjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMWExMTk2NTU1MTdmNjQyY2YzMTc0YWZlNmFhYzJjYmY2ZDEwMTI1YTkwYzg1N2ZkYzU0MjBiZTY0MDEzMzU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BzXNyPtYPlpcPfDUBytCkBrS4JIjCyGwwNSNUMG_xXU) port 443 is excluded on my PC
> 
> ![Try to Delete Exclusion](https://private-user-images.githubusercontent.com/167651924/324352009-99c8038e-3bca-42b2-8ee6-5a5f01a085b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxNzIxMjYsIm5iZiI6MTcxNDE3MTgyNiwicGF0aCI6Ii8xNjc2NTE5MjQvMzI0MzUyMDA5LTk5YzgwMzhlLTNiY2EtNDJiMi04ZWU2LTVhNWYwMWEwODViNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNlQyMjUwMjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDE3NjkyYWIxYmNiMmZlODY0YWNjMTRlOGMyNmI4Njc5ZWJmNGJjZDljZmMwMTE3MjY5MmU2ZWJjMDhiM2VmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CVORja7xvoR5AHOGPKiA4WX51OOx8dCEJkmmFaG6dYE) But when trying to delete the exclusion, I'm being denied.
> 
> could that be the problem?
> 
> Thanks

Having exactly same problem still no solution. Btw, how did you execute to get those strings (which command used under CMS)?

**Comment by derekhe** [comment]
I have no idea.

**Comment by fredo340** [comment]
ok, here is what I found on my system. I have windows 11 Pro. The RpcSs(Remote Procedure Call(RPC) Service) is using port 443 and there is no way for me to disable it or changed the port. And from what I read it is highly importantly recommended not to try to disable it. I went in safe mode with network to try to run Map enhance and it actually worked as port 443 was free. Means that it confirmed that because port 443 is occupied on my system the program wont work. Is there a way you can change the program to use another port instead of 443?

**Comment by derekhe** [comment]
In default windows 11 installation it shouldn't occupy 443 port. 
This application needs 443 port because it hijacks url calls from MSFS where I can't change the port.

**Comment by ozcan181** [comment]
I am still struggling to get over the Backend server start failed related error.  Done all th recommended points but no result.
Here below the screenshot from the codes run on CMD. Could you please advise what is wrong here?
![2024-05-03_22-40-23](https://github.com/derekhe/msfs2020-map-enhancement/assets/112335122/7a750d75-049f-4628-842e-e52c79ee68be)

**Comment by ozcan181** [comment]
> Hi, I'm having the following issues. I have disable my antivirus, reinstall, followed the FAQ. but still the same. ![ERROR](https://private-user-images.githubusercontent.com/167651924/324206740-26d17ba0-c6af-47d4-b456-c8e69a04569f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NDc5NTAsIm5iZiI6MTcxNDc0NzY1MCwicGF0aCI6Ii8xNjc2NTE5MjQvMzI0MjA2NzQwLTI2ZDE3YmEwLWM2YWYtNDdkNC1iNDU2LWM4ZTY5YTA0NTY5Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QxNDQ3MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZTY0ZTNhYjkyMWY1ZWY2OWJmZTViNWIwNTM3YjY2MjI5NjBmZDllYmVlN2EyY2E4ODU0YzU2NzAyZTE1ZGQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XC7v-t9xHuKe8mUVMBAOJKMOFLfSEKxsGN_vKTbo5II) ![Command Prompt](https://private-user-images.githubusercontent.com/167651924/324206751-d6e0b60a-9e43-4f44-9c70-e7fb2d3ffd8a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NDc5NTAsIm5iZiI6MTcxNDc0NzY1MCwicGF0aCI6Ii8xNjc2NTE5MjQvMzI0MjA2NzUxLWQ2ZTBiNjBhLTllNDMtNGY0NC05YzcwLWU3ZmIyZDNmZmQ4YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QxNDQ3MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTViZWU3YTc3NDJkYjg5YWFhMGY1YmM2MGI4NDUxMjk2ZTExY2E4Zjk1NzEzNThhNTM0ZjFmODE4ZjI0ODVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._FNu_iUDffhJtX7qN9XCHEYY9ZvLxKBuX2otqdvvZMQ) please help...

Did you find any solution?

**Comment by derekhe** [comment]
From remote log it says the 443 port is occupied. Please fix it.

**Comment by BurnleyNuts** [comment]
Hi I've had the same issue as those above since around version 7.5.3 or earlier so for about 6 months I havent been able to use this map enhancement even though I paid for it. I tried all of the suggested solutions and also reported this to Derek a while ago but no resolution so I just didn't use this for a long while. I have periodically updated the map enhancement software as new versions came along in the hope it would solve my problem but no joy. Then this morning I simply tried the App once again without doing anything of the recopmended suggestions and it just simply loaded and worked. I was prior to this morning on version 7.5.15 I've now just updated to v7.5.18 and still working fine.

**Comment by ozcan181** [comment]
> Hi I've had the same issue as those above since around version 7.5.3 or earlier so for about 6 months I havent been able to use this map enhancement even though I paid for it. I tried all of the suggested solutions and also reported this to Derek a while ago but no resolution so I just didn't use this for a long while. I have periodically updated the map enhancement software as new versions came along in the hope it would solve my problem but no joy. Then this morning I simply tried the App once again without doing anything of the recopmended suggestions and it just simply loaded and worked. I was prior to this morning on version 7.5.15 I've now just updated to v7.5.18 and still working fine.

Ohh good on you  and wish  I could get that luck too, On my side still same,  getting error message and  whatever done useless simply... 
I am giving up  sadly....

**Comment by BurnleyNuts** [comment]
Yeah, it's very strange because I guess for some people now it's nothing to do with the software release version nor any of those recommended suggestions for it to work. I have no answers sadly but mine simply started working and I did nothing, I've still got the same Norton Antivirus plus Windows Defender and I've changed nothing recently other than done Windows updates and just general maintenance. Stick with it, it might just release itself.
